jaxrs/src/main/java/org/killbill/billing/jaxrs/json/OverdueConditionJson.java

```diff
@@ -57,7 +58,9 @@ public OverdueConditionJson(@JsonProperty("timeSinceEarliestUnpaidInvoiceEqualsO
     }
 
     public OverdueConditionJson(final OverdueCondition overdueCondition) {
-        this.timeSinceEarliestUnpaidInvoiceEqualsOrExceeds = new DurationJson(overdueCondition.getTimeSinceEarliestUnpaidInvoiceEqualsOrExceeds());
+        this.timeSinceEarliestUnpaidInvoiceEqualsOrExceeds = overdueCondition.getTimeSinceEarliestUnpaidInvoiceEqualsOrExceeds() == null ?
```

Comment:
⚠️ I think checking the null value of `overdueCondition#getTimeSinceEarliestUnpaidInvoiceEqualsOrExceeds()` makes a lot of sense otherwise the code will NPE inside the `DurationJson ` CTOR; however, I don't think providing a default of one day is good, you are essentially changing the condition. Instead, I think we should default to `this.timeSinceEarliestUnpaidInvoiceEqualsOrExceeds = null` (but to be tested).

Reply:
Also, I am wondering what triggered this change - if you run your test below, the code never goes though this code as the `jaxrs` layer is our api/endpoint layer, and any test inside the overdue module will not exercise this code.